custom_components/tahoma/climate_devices/atlantic_electrical_towel_dryer.py

```diff
@@ -97,9 +97,10 @@ async def async_set_hvac_mode(self, hvac_mode: str) -> None:
     @property
     def preset_mode(self) -> Optional[str]:
         """Return the current preset mode, e.g., home, away, temp."""
-        return TAHOMA_TO_PRESET_MODE[
-            self.executor.select_state(IO_TOWEL_DRYER_TEMPORARY_STATE_STATE)
-        ]
+        state = self.executor.select_state(IO_TOWEL_DRYER_TEMPORARY_STATE_STATE)
```

Comment:
Maybe we should just not add `SUPPORT_PRESET_MODE` when this state is not available?

Reply:
Can we do that in another PR? We will have to apply this logic to other devices too.

Reply:
Why? Is that not only applicable to this device? I think the other climate devices always have the state present.